docs/source/legate-boost.rst

```diff
@@ -0,0 +1,441 @@
+.. _legate-boost:
+
+
+=============
+legate-boost
+=============
+
+This article assumes familiarity with the basic usage of gradient boosting
+libraries such as XGBoost or LightGBM, as well as ``cupynumeric`` for GPU-accelerated
+array computations. In this tutorial, these libraries are used for efficient model
+training, large-scale data handling, and accelerating computations across CPUs and GPUs.
+
+What is legate-boost?
+=====================
+
+In scenarios where high-performance training is needed across large
+datasets or distributed hardware, ``legate-boost`` offers a scalable
+alternative. ``legate-boost`` is an advanced gradient boosting library built
+on the ``Legate`` and Legion parallel programming frameworks. Unlike
+traditional boosting libraries such as XGBoost or LightGBM, ``legate-boost``
+provides a unified infrastructure that seamlessly scales across CPU's and
+GPU's, supporting both single-node and distributed training while
+integrating naturally with ``cupynumeric`` workflows for efficient
+end-to-end data processing. It enables users to define not only
+conventional boosted decision trees but also hybrid ensembles combining
+trees, kernel ridge regression, linear models, or neural networks, all
+written in Python with minimal code changes.
+
+These models are automatically parallelized and executed efficiently
+without manual data movement or partitioning. ``legate-boost`` emphasizes
+architectural simplicity, extensibility, and performance, delivering
+state-of-the-art results on tabular data while leveraging the full
+computational power of modern heterogeneous hardware.
+
+Please refer to `Distributed Computing with cupynumeric`_
+and `legate boost`_ for more
+information and detailed instructions on installation and setup.
+
+.. _Distributed Computing with cupynumeric: https://github.com/NVIDIA/accelerated-computing-hub/blob/main/Accelerated_Python_User_Guide/notebooks/Chapter_11_Distributed_Computing_cuPyNumeric.ipynb
```

Comment:
This link will 404 if somebody moves the file in this repo. So we either need to check the links don't 404 in CI somehow or link to something that is more likely to stay put.